README.md

```diff
@@ -30,3 +31,34 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+// Worked with Jacob Johnson
+
+$$ T(n) =
+   \begin{cases}
+       1 & n \leq 1\\
+       3T(\frac{n}{3}) + n^5 & n > 1
+   \end{cases}
+$$
+
+$T(n) = 3T(\frac{n}{3}) + n^5 $
+
+$= 3(3T(\frac{n}{9}) + \frac{n^5}{3^5}) + n^5$
+
+$= 9T(\frac{n}{9}) + \frac{n^5}{3^4} + n^5$
+
+$= 27T(\frac{n}{27}) + \frac{n^5}{9^4} + \frac{n^5}{3^4} + n^5$
```

Comment:
Don't think your sum is correct here. I wouldn't be able to get any of your terms above with it.